File: holiday_5.py
def sea_sick(sea):
    count_change = 0
    total = len(sea)
    for x in range(len(sea)-1):
        if sea[x] != sea[x+1]:
            count_change += 1
    perC = count_change / total
    if perC > 0.20:
        return 'Throw Up'
    else:
        return 'No Problem'

File: holiday5.js
function seaSick(x){
    let countChange = 0;
    let total = x.length;
    for (let i = 0; i < x.length-1; i++){
      if (x[i] !=  x[i+ 1]){
        countChange++;
      }
    }
    let perC = countChange / total;
    if (perC > 0.20){
      return 'Throw Up';
    }else{
      return 'No Problem';
    }
  }